src/formatDate.js

```diff
@@ -50,7 +50,38 @@
  */
 
 function formatDate(date, fromFormat, toFormat) {
-  // write code here
+  const dateInput = date;
+  const oldFormat = fromFormat;
+  const newFormat = toFormat;
+  const oldDelimiter = oldFormat.splice(3, 1);
```

Comment:
```suggestion
  const oldDelimiter = oldFormat[3];
```
Splice is a mutating method. You shouldn't mutate the origin array.
Just get access to the needed element using `[]` or `at` method